syncmaster/schemas/v1/transfers/__init__.py

```diff
@@ -255,8 +255,10 @@ def validate_increment_by(cls, values):
         source_type = values.source_params.type
         increment_by = values.strategy_params.increment_by
 
-        if source_type in FILE_CONNECTION_TYPES and increment_by != "modified_since":
-            raise ValueError("Field 'increment_by' must be equal to 'modified_since' for file source types")
+        if source_type in FILE_CONNECTION_TYPES and increment_by not in ("modified_since", "file_name"):
```

Comment:
Maybe we should rename `modified_since` -> `file_modified_since`, for consistency